astarte/device/device.py

```diff
@@ -97,17 +100,17 @@ def __init__(self,
         self.__pairing_base_url = pairing_base_url
         self.__persistency_dir = persistency_dir
         self.__credentials_secret = credentials_secret
-        self.__jwt_token = None
+        self.__jwt_token: Optional[str] = None
         self.__is_crypto_setup = False
         self.__introspection = Introspection()
         self.__is_connected = False
         self.__loop = loop
         self.__ignore_ssl_errors = ignore_ssl_errors
 
-        self.on_connected = None
-        self.on_disconnected = None
+        self.on_connected: Optional[Callable[[Device], None]] = None
+        self.on_disconnected: Optional[Callable[[Device, int], None]] = None
         self.on_aggregate_data_received = None
```

Comment:
Why this callback does not have hinting?

Reply:
This callback is not actually used inside all the SDK and is planned to be removed in a separate PR